install_mpremote.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Install mrequests to a MicroPython board using mpremote
+
+MODULES=('defaultdict.py' 'mrequests.py' 'urlencode.py' 'urlparseqs.py' 'urlunquote.py')
+DESTDIR=":/lib"
+
+# Create the root lib folder
+# Will generate an error in the output if it already exists
+# 
+# Traceback (most recent call last):
+#   File "<stdin>", line 2, in <module>
+# OSError: [Errno 17] EEXIST
+mpremote mkdir "${DESTDIR}"
```

Comment:
`mpremote` should really implement `mkdir -p`. It is possible to implement it with `mpremote exec`, but it's cumbersome.

Reply:
> It is possible to implement it with mpremote exec, but it's cumbersome.

Do you mean by executing a whole `try/except` sequence as follows?

`mpremote exec "import os\ntry:\n\tos.mkdir('lib')\nexcept Exception as e:\n\tprint('cannot create lib folder')\n"`

Not pretty, but valid. Unfortunately while that `exec()` works in a REPL, `mpremote` decided to do something with the string before injecting it and it fails.
I've asked one of ours to look into it, he may have bumped into the issue ;)

Reply:
I have filed an issue related to how `exec` breaks the input string by adding an extra `\`.
I think this is a valid option, but for now, unless we don't want to have a whole `.py` to execute as a script via `mpremote` we'll have to stick to the code returning the error if the folder exists

ref: https://github.com/micropython/micropython/issues/13693

Reply:
Yeah, I was only able to `exec` a snippet containing newlines using `mpremote` as a library from Python (see lines 32-79 in `install.py` in #14).

For now we'll have to live with the error message, I guess.